mopidy/core/tracklist.py

```diff
@@ -433,6 +434,13 @@ def add(self, tracks=None, at_position=None, uri=None, uris=None):
         tl_tracks = []
 
         for track in tracks:
+            if(self.get_length() >=
+                    self.core._config['core']['max_tracklist_length']):
+
+                raise exceptions.TracklistFull(
+                    'TracklistFull: Tried to add ' +
```

Comment:
Minor nitpicks:

- Having the exception name in the msg is probably overkill
- The `+` for string concatenation isn't needed
- s/uris/URIs/
- Message sounds a bit like you added to many URIs at once, not that the queue is full

We'll update the message after merging it.